addons/medical/ACE_Settings.hpp

```diff
@@ -59,6 +59,16 @@ class ACE_Settings {
         typeName = "SCALAR";
         values[] = {"Disabled", "50/50", "Enabled"};
     };
+    class GVAR(autoWakeUp) {
```

Comment:
This setting is not necessary, simply have the wake up time of `-1` disable this